tools/matchcompiler.py

```diff
@@ -160,7 +160,7 @@ def _compilePattern(self, pattern, nr, varid,
                 ret += '        ' + returnStatement
 
             # a|b|c
-            elif tok.find('|') > 0:
+            elif '|' in tok:
```

Comment:
this looks a bit suspicious. however the travis job should detect if this cause problems.